draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -308,7 +310,7 @@ CHAR (any {{USASCII}} character), VCHAR (any visible {{USASCII}} character),
 and WSP (whitespace).
 
 The OWS (optional whitespace) and BWS (bad whitespace) rules are defined in
-Section 5.6.3 of {{RFC9110}}.
+{{Section 5.6.3 of RFC9110}}.
```

Comment:
Huh. I didn't realize that worked. :)

Reply:
It's a nice feature. I was even more surprised that line 1519 worked
`(see {{Sections 15.4.2 and 15.4.3 of RFC9110}});`